src/Core/Localization/Currency/CurrencyData.php

```diff
@@ -136,6 +148,13 @@ public function overrideWith(CurrencyData $currencyData)
             $this->names = array_merge($this->names, $currencyData->getNames());
         }
 
+        if ($currencyData->getPatterns() !== null) {
+            if (null === $this->patterns) {
+                $this->patterns = [];
+            }
+            $this->patterns = array_merge($this->patterns, $currencyData->getPatterns());
```

Comment:
And you can remove the condition above
```suggestion
            $this->patterns = array_merge($this->patterns ??  [], $currencyData->getPatterns());
```

Reply:
Not bad!